### PR TITLE
Feature/reptiles

### PR DESCRIPTION
**Pull request by sabawaheed27**
I added global css and finished my reptiles page

### CODE REVIEW

File: js/reptiles.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
+
+function Animal(image, name, group, lifespan, weight, food, length, found, description) {
```
Comment (SSSpac) [inline js/reptiles.js:2]: nice work with the animals object !

**Review by SSSpac — APPROVED** [review]

**Review by yevhenii-kapustian — APPROVED** [review]

**Review by Maatala — APPROVED** [review]
looks good, nice!

File: js/reptiles.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+  
+  const onClick = {
+    lastClicked: null,
+    showWelcomeMsg: false,
+  }
+  
+  const showAnimal = (animalName) => {
+    const animalInfo = document.getElementById("content");
+    let animal = getAnimal(animalName);
+
```
Comment (andrewkalumba) [inline js/reptiles.js:35]: alot of space in between

File: js/reptiles.js [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+  const items = document.querySelectorAll('li');
+  items.forEach(item => {
+    item.addEventListener('click', (e) => {
+      showAnimal(e.target.textContent);
+  
+    }
+    )
+  })
+  
+
```
Comment (andrewkalumba) [inline js/reptiles.js:91]: remove some space

File: stylesMain/reptiles.css [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+  border: 2px solid rgb(120, 115, 115);
+}
+
+.sidebar {
+  flex: 0 0 25%;
+  font-weight: bold;
+  padding-top: 50px;
+  font-style: normal;
+  margin: 16px;
+
```
Comment (andrewkalumba) [inline stylesMain/reptiles.css:17]: can remove this space

**Review by andrewkalumba — COMMENTED** [review]
Fix the little errors i commented on otherwise the work looks good and you can proceed with merging into develop branch

**Review by andrewkalumba — APPROVED** [review]
fix the little errors i commented on otherwise the work looks great and you can merge into develop after the fix

File: js/reptiles.js (hunk 3 above)
Comment (sabawaheed27) [reply js/reptiles.js:91]: I'm very careful abut spaces, may be it seems here. will double check

File: js/reptiles.js (hunk 2 above)
Comment (sabawaheed27) [reply js/reptiles.js:35]: will recheck . Thanks

**Pull request merged**